dashboard/lti_new.py

```diff
@@ -234,6 +235,18 @@ def extract_launch_variables_for_tool_use(request, message_launch):
         password = ''.join(random.sample(string.ascii_letters, settings.RANDOM_PASSWORD_DEFAULT_LENGTH))
         user_obj = User.objects.create_user(username=username, email=email, password=password, first_name=first_name,
                                             last_name=last_name)
+
+    # Add username into the MyLA User table, since the data was not pulled in from cron job
+    user_id = CANVAS_DATA_ID_INCREMENT + int(canvas_user_id)
+    try:
+        myla_user_obj = MyLAUser.objects.get(user_id=user_id)
+        # update
+        myla_user_obj.name = full_name
+        myla_user_obj.sis_name = username
+        myla_user_obj.save()
+    except MyLAUser.DoesNotExist:
+        logger.warn(f'Cannot find MyLA user {username}')
```

Comment:
I need to test still, but I think this will fail if you have a user with multiple enrollments, maybe with a `MultipleObjectsReturned` exception?

We probably need to use `.update(...)`: https://docs.djangoproject.com/en/4.1/ref/models/querysets/#update

Reply:
Confirmed:
```
[ERROR] [lti_new.py:56] LTI error: get() returned more than one User -- it returned 2!
```

Reply:
addressed the problem in [6ad1ed2](https://github.com/tl-its-umich-edu/my-learning-analytics/commit/6ad1ed26ddc4270862b2b8e8d666bc237dd7b7c2)